types/mapbox-gl/mapbox-gl-tests.ts

```diff
@@ -805,6 +805,22 @@ map.showPadding = false;
 expectType<mapboxgl.Map>(map.setFilter('layerId', true));
 expectType<mapboxgl.Map>(map.setFilter('layerId', false));
 
+expectType<mapboxgl.Map>(map.setMinZoom(5));
+expectType<mapboxgl.Map>(map.setMaxZoom(10));
+
+// $ExpectType number
+map.getMinZoom();
+// $ExpectType number
+map.getMaxZoom();
+
+expectType<mapboxgl.Map>(map.setMinPitch(5));
```

Comment:
Why is expectType<> used here instead of $ExpectType? I think it's a good idea to use $ExpectType for newly added tests.